docs/theming-guide.adoc

```diff
@@ -4226,9 +4226,6 @@ IMPORTANT: If you don't specify a height for either the header or footer key, it
 
 If you define running header and footer content in your theme (including the height), you can still disable this content per document by setting the `noheader` and `nofooter` attributes in the AsciiDoc document header, respectively.
 
-If content is not specified for the running footer, the page number (i.e., `\{page-number}`) is shown on the left on verso pages and the right on recto pages.
```

Comment:
Even though it's no longer explicitly added in `converter.rb`, I guess any theme extending base or default will still see this "default page numbers" behaviour?
Perhaps it's still worth documenting e.g. `footer_verso_left_content: null` ? :man_shrugging:

Reply:
Sure. I've gone ahead and updated the docs with this information.